### DISCUSSION TITLE
Question related to output from findfs

### DISCUSSION THREAD

**Discussion by enool** [util-linux/util-linux#2914] [Q&A] [answered]
Hey, I'm wondering if it is safe to depend on findfs to return the input as is when no tag is specified?

This didn't have any explicit mention in man pages but seems to exist at least in versions 2.4, 2.37.2 and 2.36.1.

> findfs doesnotexist
> doesnotexist

Printing the input as is could be quite useful if you have some values that are a union of <path, tag=value>. This line from man pages makes me slightly nervous though:

> If the filesystem or partition is found, the device name will be
> printed on stdout.

Bug or a feature? :thinking:

**Comment by karelzak** [comment, marked as answer]
The man page is inaccurate. Fixed by https://github.com/util-linux/util-linux/commit/07aacb371470b5561f13b95e399e5ff77e417b8fe. 

The current behavior is based on libblkid API, so I think it's safe to rely on this :-)
